content/verification/veteran-verification/release-notes/2023-09-20

```diff
@@ -0,0 +1,11 @@
+Additional fields have been added to the disability rating endpoint as follows;
```

Comment:
Are these additional fields just for v2 of the Veteran Service History and Eligibility API?

Reply:
Yes, just for V2